src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Jesus.java

```diff
@@ -215,9 +215,9 @@ else if ((event.fluidState.getFluid() == Fluids.LAVA || event.fluidState.getFlui
     private void onFluidCollisionShape(CollisionShapeEvent event) {
         if (event.state.getFluidState().isEmpty()) return;
 
-        if (event.state.getFluidState().isOf(Fluids.WATER) && !mc.player.isTouchingWater() && waterShouldBeSolid()) {
+        if (event.state.getBlock() == Blocks.WATER && !mc.player.isTouchingWater() && waterShouldBeSolid()) {
```

Comment:
```suggestion
        if (event.state.getBlock() == Blocks.WATER && !mc.player.isTouchingWater() && waterShouldBeSolid()) {
```
Is the fluid tag/function obsolete, will it not be used?

Reply:
No idea if it's obsolete but it doesn't work properly for the portions of liquid that are flowing other than the source block

Reply:
This is a video showing how it acts without the code change. (Broken)
https://github.com/MeteorDevelopment/meteor-client/assets/115842502/eb3c48ba-9197-4e3b-86db-fe7429483cda
This is how it acts, after that smol change. (Fixed)
https://github.com/MeteorDevelopment/meteor-client/assets/115842502/bbef313e-dd35-428a-82a0-1ada241227c6